pdns/dnsdistdist/dnsdist-selectors-definitions.yml

```diff
@@ -48,7 +48,7 @@
       rust-type: "String"
       description: "The full 16bit RCode will be matched. If no EDNS OPT RR is present, the upper 12 bits are treated as 0"
 - name: "HTTPHeader"
-  description: "Matches DNS over HTTPS queries with a HTTP header name whose content matches the supplied regular expression. It is necessary to set the ``keepIncomingHeaders`` to :func:`addDOHLocal()` to use this rule"
+  description: "Matches DNS over HTTPS queries with a HTTP header name whose content matches the supplied regular expression in Posix Extended Regular Expressions format. It is necessary to set the ``keepIncomingHeaders`` to :func:`addDOHLocal()` to use this rule"
```

Comment:
```suggestion
  description: "Matches DNS over HTTPS queries with a HTTP header name whose content matches the supplied regular expression in Posix Extended Regular Expressions format. The match is done in a case-insensitive way. It is necessary to set the ``keepIncomingHeaders`` to :func:`addDOHLocal()` to use this rule"
```

Reply:
I'm not 100% with this text addition. Once we have a proper text, this also applies to the other cases

Reply:
Yes please.